validation/reproschema-shacl.ttl

```diff
@@ -206,11 +222,17 @@ reproschema:ChoicesShape a sh:NodeShape ;
     sh:property [ sh:nodeKind sh:IRI ;
             sh:path schema:image ],
 
+        [ sh:nodeKind sh:IRI ;
```

Comment:
why is this back here? Dontknow and skipped are not properties of choice, they are potential values, no?

Reply:
sorry, this happened by mistake!